pom.xml

```diff
@@ -119,7 +119,7 @@
         <maven-jar-plugin.version>3.2.0</maven-jar-plugin.version>
         <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
         <!-- NOTE: We pin to this version due to https://github.com/apache/camel-quarkus/issues/723 -->
```

Comment:
I think we can safely remove the NOTE comment now

Reply:
I'll do it